drivers/hmis/app/models/hmis/ce/match/sql_expression_translator.rb

```diff
@@ -11,7 +11,12 @@ class SqlExpressionTranslator
 
     def self.call(expression, field_map)
       calculator = Hmis::Ce::Match::CalculatorFactory.build
-      ast = calculator.ast(expression)
+      begin
+        ast = calculator.ast(expression)
+      rescue Dentaku::Error => e
+        err_with_context = "Error parsing expression '#{expression}': #{e.message}"
+        raise $ERROR_INFO, err_with_context, $ERROR_INFO.backtrace
```

Comment:
Do we have to use the global ERROR_INFO here?
```suggestion
        raise e, err_with_context, e.backtrace
```